packages/mui-base/src/ButtonUnstyled/ButtonUnstyled.tsx

```diff
@@ -152,10 +152,6 @@ ButtonUnstyled.propTypes /* remove-proptypes */ = {
    * @default false
    */
   disabled: PropTypes.bool,
-  /**
-   * @ignore
-   */
-  onClick: PropTypes.func,
```

Comment:
is this removed by the script?

Reply:
Yes, yarn proptypes removed it. When working on the Select I've noticed an issue on how events are handled in ButtonUnstyled and will improve it in a separate PR.